### PR TITLE
Team2 task20 patch1

### PR DESCRIPTION
**Pull request by Soncity2**
updated task 20 + team readme

### CODE REVIEW

File: src/team_2/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Team 2 - Son and Gal
+
+### Tasks:
+
+| Task ID | Description                                                        | Status                                   | Progress                        |
```
Comment (RyanWri) [inline src/team_2/README.md:5]: write conclusions instead, the project already track the issues

File: src/team_2/task_20/hDoctor_columns.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (RyanWri) [inline src/team_2/task_20/hDoctor_columns.json]: move translation files to translations folder (create one if you don't have)

File: src/team_2/task_9/task_9.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
+import os
```
Comment (RyanWri) [inline src/team_2/task_9/task_9.py]: duplicate code, import function from relevant packages (team 1 i.e)

**Review by RyanWri — CHANGES_REQUESTED** [review]
Please review my comments

**Comment by Soncity2** [comment]
Edited the Requested changes, wasn't able to run the code because of updated repo not pulled

**Review by RyanWri — APPROVED** [review]
Approved, just solve your conflicts before merge please

**Pull request merged**